### PR TITLE
Add MilkyWay

### PR DESCRIPTION
**Pull request by RiccardoM**
This PR adds the upcoming `milkyway` chain to the registry.

### CODE REVIEW

File: milkyway/assetlist.json [hunk 1]
```diff
@@ -0,0 +1,291 @@
...
+          "aliases": [
+            "umilktia"
+          ]
+        },
+        {
+          "denom": "milkTIA",
+          "exponent": 6
+        }
+      ],
+      "type_asset": "sdk.coin",
```
Comment (JeremyParish69) [inline milkyway/assetlist.json:124]: this would be ics20

File: milkyway/chain.json [hunk 2]
```diff
@@ -0,0 +1,154 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "milkyway",
+  "status": "live",
+  "network_type": "mainnet",
+  "website": "https://milkyway.zone/",
+  "pretty_name": "MilkyWay",
+  "chain_type": "cosmos",
+  "chain_id": "milkyway",
+  "bech32_prefix": "milk",
```
Comment (JeremyParish69) [inline milkyway/chain.json:10]: Please register this prefix
https://github.com/satoshilabs/slips/blob/master/slip-0173.md

File: _IBC/milkyway-osmosis.json [hunk 3]
```diff
@@ -0,0 +1,32 @@
+{
+  "$schema": "../ibc_data.schema.json",
+  "chain_1": {
+    "chain_name": "milkyway",
+    "client_id": "07-tendermint-1",
+    "connection_id": "connection-0",
```
Comment (JeremyParish69) [inline _IBC/milkyway-osmosis.json:6]: Extra comma here

Comment (manu0466) [reply _IBC/milkyway-osmosis.json]: Fixed!

**Comment by manu0466** [comment]
@JeremyParish69 should now pass all the tests.

**Pull request merged**